B-team-progathon/src/main/java/frc/robot/subsystem/Claw.java

```diff
@@ -1,5 +1,79 @@
 package frc.robot.subsystem;
 
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import edu.wpi.first.wpilibj.Solenoid;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+
+// I WILL REPLACE MAGIC NUMBERS WITH CONSTANTS FILE LATER
+// Pneumatics HUB and Controller stuff??
+
+enum ClawStates {
+    IDLE,
+    OUTPUTING,
+    INTAKING,
+    HOLDING
+}
+
 public class Claw {
+    // Solenoids for claw open and close
+    private Solenoid solenoidLeft;
+    private Solenoid solenoidRight;
+    
+    // NEO 550 motors for wheel spinning
+    private CANSparkMax motorLeft;
+    private CANSparkMax motorRight;
+
+    private ClawStates state;
+
     
+    public Claw() {
+        // Initialize solenoids
+        solenoidLeft = new Solenoid(PneumaticsModuleType.CTREPCM, 0); // NOT ACCURATE
+        solenoidRight = new Solenoid(PneumaticsModuleType.CTREPCM, 1); // NOT ACCURATE
+        
+        // Initialize NEO 550 motors
+        motorLeft = new CANSparkMax(2, MotorType.kBrushless); // NOT ACCURATE
+        motorRight = new CANSparkMax(3, MotorType.kBrushless); // NOT ACCURATE
```

Comment:
instead of individually setting the values of both motors you can invert one motor and then make it follow the other motor. Should be motorLeft.follow(motorRight); and motorRight.setInverted(True); or something similar to that. Check the rev java docs for specifics (google rev java docs)